src/classes/STG_InstallScript.cls

```diff
@@ -250,4 +252,20 @@ global without sharing class STG_InstallScript implements InstallHandler {
         }
         return null;
     }
+
+    /*******************************************************************************************************
+    * @description Disables Preferred Email handling when upgrading from 1.30
+    * @return void
+    */
+    global static void disablePreferredEmailEnforcement(Version previousVersion) {
```

Comment:
Ugh, yes. The "disable" meaning the validation is enabled got me. Thanks for the clarification.